lib/dynflow/clock.rb

```diff
@@ -104,8 +104,12 @@ class ClockReference < Concurrent::Actor::Reference
     def ping(who, time, with_what = nil, where = :<<)
       Type! time, Time, Numeric
       time  = Time.now + time if time.is_a? Numeric
```

Comment:
Is it ok to use local time if we may send it beyond the boundary of the current process?

Reply:
It's converted to string with timezone. So parsing it will read the timezone. No issue there. I would be more worried about precision....